HLTrigger/Tools/scripts/convertToRaw

```diff
@@ -199,7 +268,42 @@ for run in sorted(content):
 
         # write the final EoR file
         # implemented by hand instead of using json.dump() to match the style used by the DAQ tools
+        assert len(converted_files) == summary['data'][1]
         eor_file = run_path + f'/run{run:06d}_ls0000_EoR.jsn'
-        f = open(eor_file, 'w')
-        f.write('{\n   "data" : [ "%d", "%d", "%d", "%d" ],\n   "definition" : "%s",\n   "source" : "%s"\n}\n' % (summary['data'][0], summary['data'][1], summary['data'][2], summary['data'][3], summary['definition'], summary['source']))
-        f.close()
+        with open(eor_file, 'w') as file:
+            file.write('{\n   "data" : [ "%d", "%d", "%d", "%d" ],\n   "definition" : "%s",\n   "source" : "%s"\n}\n' % (summary['data'][0], summary['data'][1], summary['data'][2], summary['data'][3], summary['definition'], summary['source']))
+            file.close()
+
+    # mark the .raw files as not executable
+    for f in converted_files:
+        os.chmod(f, 0o644)
+
+    # write a cff file for processing the converted files
+    cff_file = args.output_directory + f'/run{run:06d}_cff.py'
+    with open(cff_file, 'w') as file:
+        file.write("""import FWCore.ParameterSet.Config as cms
+
+from EventFilter.Utilities.FedRawDataInputSource_cfi import source as _source
+source = _source.clone(
+    eventChunkSize = 200,   # MB
+    eventChunkBlock = 200,  # MB
+    numBuffers = 4,
+    maxBufferedFiles = 4,
+    fileListMode = True,
+    fileNames = (
+%s
+    )
+)
+
+from EventFilter.Utilities.EvFDaqDirector_cfi import EvFDaqDirector as _EvFDaqDirector
+EvFDaqDirector = _EvFDaqDirector.clone(
+    buBaseDir = '%s',
+    runNumber = %d
+)
+
+from EventFilter.Utilities.FastMonitoringService_cfi import FastMonitoringService as _FastMonitoringService
+FastMonitoringService = _FastMonitoringService.clone()
+""" % ('\n'.join("        '" + f + "'," for f in converted_files), args.output_directory, run))
+        file.close()
```

Comment:
I thought with-open-as takes care of closing the file (so `close` wouldnt be needed).